packages/content/types/index.d.ts

```diff
@@ -1,4 +1,5 @@
 import '@nuxt/types'
+import '@nuxt/types/config/hooks'
```

Comment:
I don't believe this import is necessary as the `@nuxt/types` also imports the config/hooks. In my setup it wasn't necessary, can be mistaken

Reply:
You might be right I didn't test this - so I wantet to be sure ;D